pkg/reconciler/route/resources/virtual_service.go

```diff
@@ -164,7 +158,6 @@ func buildHTTPRoute(namespace, urlPath string, appNames []string) ([]networking.
 		return []networking.HTTPRoute{
 			{
 				Match: pathMatchers,
-				Route: buildRouteDestination(),
 				Fault: &networking.HTTPFaultInjection{
```

Comment:
Does this still produce 503s with an unbound route if we don't have the route here?

Reply:
added an integration test to check for 503